src/main/lib/utils.js

```diff
@@ -1,8 +1,12 @@
 import path from 'path';
 
 export function getPathToFile(fileName) {
+  return `file://${getURIPathToFile(fileName)}`;
+}
+
+export function getURIPathToFile(fileName) {
```

Comment:
This should be `getPathToFile` as it returns a regular file path.

Reply:
k